<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiveAddressInfo</class>
 <widget class="QDialog" name="ReceiveAddressInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Request info to </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_lQrCode">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="m_teInfo">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff5500;&quot;&gt;афывафыва&lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#ff5500;&quot;&gt;фывафы&lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;вафыва&lt;/span&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;фыва&lt;/span&gt;&lt;span style=&quot; font-weight:600; font-style:italic; text-decoration: underline;&quot;&gt;фывафыва&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_btnCopyUri">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Copy URI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnCopyAddress">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Copy Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnSaveImage">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Save image ...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnClose">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="m_actSaveImage">
   <property name="text">
    <string>Save image ...</string>
   </property>
   <property name="toolTip">
    <string>Save image ...</string>
   </property>
  </action>
  <action name="m_actCopyImage">
   <property name="text">
    <string>Copy image</string>
   </property>
   <property name="toolTip">
    <string>Copy image</string>
   </property>
  </action>
  <action name="m_actSelectAll">
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="toolTip">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="m_actCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_btnClose</sender>
   <signal>clicked()</signal>
   <receiver>ReceiveAddressInfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
